helm/ffc-pay-batch-processor/Chart.yaml

```diff
@@ -5,5 +5,5 @@ name: ffc-pay-batch-processor
 version: 1.0.0
 dependencies:
 - name: ffc-helm-library
-  version: ^2.11.0
+  version: ^3.4.0
```

Comment:
Remove caret

Reply:
What this is acc'ly saying is, "Use the newer, updated Helm library version 3.4.0 rather than 2.11.0"

In tht new release of this we (I say, "we"... John) made some simplifications by adding some default values to things that otherwise we would have to define.

Ask is to find which values in `values.yaml` are no longer needed because shiny, new v3.4.0 defaults these values to want we want.

Start of the breadcrumb trail, try looking in the `_container.yaml` template file here: https://github.com/DEFRA/ffc-helm-library/tree/master/ffc-helm-library/templates